setup.py

```diff
@@ -31,6 +31,7 @@
             "pdoc",
             "pre-commit",
             "types-python-dateutil",
+            "maturin>=1.4,<2.0",
```

Comment:
Perhaps we can add a new entry in `extras_require` and move this there, e.g. `dev_rs`?

Reply:
Why?

Reply:
If we plan to run the entire test suite against, why shouldn't it be a part of `dev`?

Reply:
ah I see, makes sense